src/stylus/fablevision/breed-button.styl

```diff
@@ -0,0 +1,27 @@
+.geniblocks.breed-button
+  background: url('../resources/fablevision/layout-c/breed_surround.png')
+  width: 370px
+  height: 369px
+  display: flex
+  justify-content: center
+  align-items: center
+
+  .clickable-button
+    width: 139px
+    height: 138px
+    display: flex
+    justify-content: center
+    align-items: center
+    background: url('../resources/fablevision/layout-c/breed_button_UNCLICKED.png')
+    cursor: pointer
+
+    &:active
+    &.disabled
+      cursor: default
+      background: url('../resources/fablevision/layout-c/breed_button_CLICKED.png')
+
+    .text
```

Comment:
Thanks for all these comments Kirk! I wrote it like this originally because I was thinking about it just from the perspective of this file, where it's clear that `text` is part of a breed button. In my new commit, I took your suggestion and moved `button-label` to its own class, and defined it in `challenges.styl` so all labels have the correct `font-family`. I can see how this approach will reduce the amount of CSS I write going forward- let me know if I have the right idea.